### PR TITLE
[11.0][IMP] stock_pack_operation_autofill. Merge stock_picking_transfer_lot_autoassign

### PR DESCRIPTION
**Pull request by sergio-teruel**
Now this module does stock_picking_transfer_lot_autoassign module fuctions.
There was a PR for migrate stock_picking_transfer_lot_autoassign to v11 here: https://github.com/OCA/stock-logistics-workflow/pull/430 so I think that PR should be closed.
cc @Tecnativa

### CODE REVIEW

**Comment by sergio-teruel** [comment]
ping @chienandalu

**Review by chienandalu — APPROVED** [review]
Tested ok :+1:

File: stock_pack_operation_auto_fill/models/stock_picking_type.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+from odoo import fields, models
+
+
+class StockPickingType(models.Model):
+    _inherit = 'stock.picking.type'
+
+    auto_fill_operation = fields.Boolean(
+        string="Auto fill operations",
+    )
+    avoid_internal_assignment = fields.Boolean(
```
Comment (pedrobaeza) [inline stock_pack_operation_auto_fill/models/stock_picking_type.py]: Call it `avoid_lot_assignment` and default to `True` for preserving the previous way or working.

File: stock_pack_operation_auto_fill/README.rst [hunk 2]
```diff
@@ -1,78 +1 @@
...
-   :target: https://odoo-community.org
-
-This module is maintained by the OCA.
-
-OCA, or the Odoo Community Association, is a nonprofit organization whose
-mission is to support the collaborative development of Odoo features and
-promote its widespread use.
-
-To contribute to this module, please visit https://odoo-community.org.
+**This file is going to be generated by oca-gen-addon-readme.**
```
Comment (pedrobaeza) [inline stock_pack_operation_auto_fill/README.rst]: Why don't you install locally maintainer-tools and run the script for generating already the README and thus you reduce the diff size? I can do it if you prefer.

File: stock_pack_operation_auto_fill/i18n/es.po [hunk 3]
```diff
@@ -1,35 +1,47 @@
...
+#: model:ir.model.fields,field_description:stock_pack_operation_auto_fill.field_stock_picking_auto_fill_operation
+#: model:ir.model.fields,field_description:stock_pack_operation_auto_fill.field_stock_picking_type_auto_fill_operation
+msgid "Auto fill operations"
+msgstr "Autocompletar operaciones"
+
 #. module: stock_pack_operation_auto_fill
 #: model:ir.ui.view,arch_db:stock_pack_operation_auto_fill.stock_picking_form_view
 msgid "AutoFill"
-msgstr ""
+msgstr "c"
```
Comment (chienandalu) [inline stock_pack_operation_auto_fill/i18n/es.po]: Autocompletar

**Comment by gurneyalex** [comment]
@sergio-teruel can you rebase an resolve the conflict?

**Comment by pedrobaeza** [comment]
@gurneyalex rebase and green

File: stock_pack_operation_auto_fill/__manifest__.py [hunk 4]
```diff
@@ -1,18 +1,22 @@
...
     'summary': """
         Stock pack operation auto fill""",
-    'version': '11.0.1.0.1',
+    'version': '11.0.2.0.0',
     'license': 'AGPL-3',
-    'author': 'ACSONE SA/NV,Odoo Community Association (OCA)',
+    'author': 'ACSONE SA/NV,'
+              'Tecnativa,'
+              'Odoo Community Association (OCA)',
     'website': 'https://acsone.eu/',
```
Comment (Nikul-Chaudhary) [inline stock_pack_operation_auto_fill/__manifest__.py]: @sergio-teruel Improve URL ser per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L10)

File: stock_pack_operation_auto_fill/views/stock_picking_type_views.xml [hunk 5]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record model="ir.ui.view" id="view_picking_type_form_assignment">
+        <field name="name">stock.picking.type.assignment</field>
+        <field name="model">stock.picking.type</field>
+        <field name="inherit_id" ref="stock.view_picking_type_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='use_existing_lots']" position="after">
```
Comment (Nikul-Chaudhary) [inline stock_pack_operation_auto_fill/views/stock_picking_type_views.xml]: @sergio-teruel IMO Directly use `field`

**Review by Nikul-Chaudhary — COMMENTED** [review]
Small Change

**Review by pedrobaeza — APPROVED** [review]
I have attended pending comments, so I merge.

**Pull request merged**